scripts/fetch_cubes.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+
+# Copyright (c) 2019 STMicroelectronics.
+# This script will update the repo on each STM32Cube
+# from the latest version on https://github.com/STMicroelectronics
+#
+# usage : 'python fetch_cubes.py'
```

Comment:
Maybe add the location from where scripts are supposed to be launched

Reply:
Done